src/WCSimConstructCylinder.cc

```diff
@@ -1319,10 +1319,13 @@ G4LogicalVolume* WCSimDetectorConstruction::ConstructCaps(G4int zflip)
   for ( int i = -CapNCell ; i <  CapNCell; i++) {
     for (int j = -CapNCell ; j <  CapNCell; j++)   {
 
-       
+      // When you use WCTE geometry, please change xoffset and yoffset. (M.Shinoki)
```

Comment:
@mshinoki Can you make a new flag for this? So instead of `WCSimDetector->SetIsNuPrism(true)` below, make a new `WCSimDetector->SetIsNuPrismBeamTest(true)` function, and then use that variable to switch here.